metazoa/whatsnew.md

```diff
@@ -1,41 +1,41 @@
-      - _Ixodes persulcatus_ (Taiga tick, GCA\_013358835.1)
-    - Data synchronised with the WormBase project
-      - Updated assemblies and gene sets
-        - _Schistosoma mansoni_
-      - Updated gene sets
-        - _Brugia malayi_
-        - _Caenorhabditis brenneri_
-        - _Caenorhabditis briggsae_
-        - _Caenorhabditis elegans_
-        - _Caenorhabditis japonica_
-        - _Caenorhabditis remanei_
-        - _Onchocerca volvulus_
-        - _Pristionchus pacificus_
-        - _Strongyloides ratti_
-      - New species
-        - _Ascaris suum_
-        - _Hymenolepis microstoma_
-        - _Trichuris muris_
-        - _Necator americanus_
-    - Updated protein features for all species using [InterProScan](http://www.ebi.ac.uk/interpro/search/sequence-search) with version 86 of [InterPro](https://www.ebi.ac.uk/interpro/)
-    - Updated BioMarts for all gene and variation data
-    - Updated pan-taxonomic gene trees and homologies
+### Updated data
+- Assembly and gene set data updated
+  - _Acyrthosiphon pisum_ (Pea aphid, GCA\_005508785.1)
+  - _Anoplophora glabripennis_ (Asian long-horned beetle, GCA\_000390285.2)
+  - _Bombus impatiens_ (Common eastern bumblebee, GCA\_000188095.4)
+  - _Bombyx mori_ (Domestic silkworm, GCA\_014905235.2)
+  - _Danaus plexippus_ (Monarch butterfly, GCA\_009731565.1)
+  - _Glossina fuscipes_ (Tsetse fly, GCA\_014805625.1)
+  - _Solenopsis invicta_ (Red fire ant, GCA\_016802725.1)
+  - _Sarcoptes scabiei_ (Itch mite, GCA\_014595675.1)
+  - _Teleopsis dalmanni_ (Malaysian stalk-eyed fly, GCA\_002237135.2)
+- New assemblies
+  - _Culex quinquefasciatus_ (Southern house mosquito, GCA\_015732765.1)
+  - _Ixodes scapularis_ (Black-legged tick, GCA\_016920785.2)
+  - _Dermacentor silvarum_ (Ticks, GCA\_013339745.1)
+  - _Rhipicephalus microplus_ (Southern cattle tick, GCA\_013339725.1)
+  - _Rhipicephalus sanguineus_ (Brown dog tick, GCA\_013339695.1)
+  - _Haemaphysalis longicornis_ (Longhorned tick, GCA\_013339765.1)
+  - _Hyalomma asiaticum_ (Ticks, GCA\_013339685.1)
+  - _Ixodes persulcatus_ (Taiga tick, GCA\_013358835.1)
+- Data synchronised with WormBase and WormBase ParaSite
+  - Updated assemblies and gene sets
+    - _Schistosoma mansoni_
+  - Updated gene sets
+    - _Brugia malayi_
+    - _Caenorhabditis brenneri_
+    - _Caenorhabditis briggsae_
+    - _Caenorhabditis elegans_
+    - _Caenorhabditis japonica_
+    - _Caenorhabditis remanei_
+    - _Onchocerca volvulus_
+    - _Pristionchus pacificus_
+    - _Strongyloides ratti_
+  - New species
+    - _Ascaris suum_
+    - _Hymenolepis microstoma_
+    - _Trichuris muris_
+    - _Necator americanus_
+  - Updated protein features for all species using [InterProScan](http://www.ebi.ac.uk/interpro/search/sequence-search) with version 86 of [InterPro](https://www.ebi.ac.uk/interpro/)
+  - Updated BioMarts for all gene and variation data
+  -a Updated pan-taxonomic gene trees and homologies
```

Comment:
I think there's a typo there. It looks like that line gets wrapped to the previous line, rather than being another bullet point.

Reply:
done. sorry :)